doc/md/examples/grammar.txt

```diff
@@ -7,11 +7,16 @@
     X
     X SEP <list(X, SEP)>
 
-<obj_sort> ::= 
+<typ_obj_sort> ::= 
     'object'
     'actor'
     'module'
 
+<obj_sort> ::= 
+    'object'
+    'persistent'? 'actor'
```

Comment:
Maybe a very brief mentioning in the docu would be good. We can then follow up with more cross-cutting docu updates later.

Reply:
Yeah, I'll do that.